dart/lib/src/sentry.dart

```diff
@@ -41,8 +41,11 @@ class Sentry {
     @internal bool callAppRunnerInRunZonedGuarded = true,
     @internal RunZonedGuardedOnError? runZonedGuardedOnError,
     @internal SentryOptions? options,
+    @internal bool devMode = false,
```

Comment:
Since `devMode` is part of `SentryOptions` already, does it make sense to still be a method param? is there a reason to do so?

Reply:
Wanted to have it set before the `optiomnsConfiguration` callback, but since we also have an `options` parameter this is not needed.